charts/portainer/values.yaml.gotmpl

```diff
@@ -18,6 +18,12 @@ serviceAccount:
   # The name of the service account to use.
   # If not set and create is true, a name is generated using the fullname template
   name: portainer-sa-clusteradmin
+persistence:
+  enabled: true
+  size: "10Gi"
```

Comment:
if this is a max size of the portainer's volume, I think `1Gi` is more then enough. It is 59MB on inhouse-master and 3.8MB on NIH STAG at the moment

Reply:
true will change thx